app/src/androidTest/java/com/swent/assos/CreateTicketTest.kt

```diff
@@ -53,4 +90,55 @@ class CreateTicketTest : SuperTest() {
       }
     }
   }
+
+  @Test
+  fun enterWrongEmail() {
+
+    run {
+      ComposeScreen.onComposeScreen<CreateTicketScreen>(composeTestRule) {
+        step("Enter wrong email") {
+          emailField {
+            assertIsDisplayed()
+            performTextInput("notjohn.doe@epfl.ch")
+          }
+        }
+        step("click") { composeTestRule.onNodeWithText("Submit").performClick() }
+
+        composeTestRule.waitForIdle()
+        // wait until test "Ok" is displayed
+        composeTestRule.waitUntil(
+            condition = { composeTestRule.onNodeWithText("Ok").isDisplayed() },
+            timeoutMillis = 5000)
+
+        step("dismiss") { composeTestRule.onNodeWithText("Ok").performClick() }
```

Comment:
I don't get how you can differentiate the success dialog with the failure dialog if you search for a node with the "Ok" text as both dialogs have that same text.

Reply:
that is why I check at the end the number of tickets in success